.travis.yml

```diff
@@ -16,7 +16,7 @@ before_install:
   - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
   - composer self-update
 
-install: travis_retry composer update --prefer-dist
+install: travis_retry composer update --prefer-dist --ignore-platform-reqs
```

Comment:
Shouldn't this be set only for nightly?

Reply:
No need. If we had extensions requirement it would make sense, but as `nightly` is failing due to the reason of being a `8.x` version, not a `7.x`, I don't see a reason for doing that.

Reply:
I'm not convinced, our deps might have some platform requirements that must be met and it'd be a pity to ignore them

Reply:
Correct. I think we should rather use `composer require "php:^7.2|^8.0" in that one specific build to fix this. `ignore-platform-reqs` is a very dangerous tool.

Reply:
Or even `php:^8.0 as 7.4.0` if that works (haven't ever tried this). It might help with dependencies that don't allow php 8 yet.

Reply:
The nightly build is still allowed to fail. I'd like to have `ignore-platform-reqs` only for PHP 8 nightly builds too.

Reply:
Ok, I'm gonna go for `|^8.0` in our `composer.json` as the tests are green and if someone, for some reason is already using that version, can safely use this package 👍

Reply:
One of our dependencies will forbid that, they will not be able to, and the build will still be failing I think. See https://packagist.org/packages/dealerdirect/phpcodesniffer-composer-installer

Reply:
See similar issue here: https://github.com/greg0ire/coding-standard/pull/1/checks?check_run_id=428224795#step:5:12

Reply:
@greg0ire Thanks for the heads up about that dependency. I'll close this one for now.

Once PHPCS dependencies start to support PHP 8/master I'll come back to this 👍

Reply:
@carusogabriel why don't you try what I recommended in https://github.com/doctrine/coding-standard/pull/156#discussion_r375136616 ? You don't like it?